### PR TITLE
Kinetic Tremor task and various Sage Design UI Updates

### PR DESCRIPTION
**Pull request by dephillipsmichael**
This PR adds the kinetic tremor task and contains various updates to the default UI of sage motor control steps.

1. The close button is now 50x50 with a smaller leading/top margin on all default and custom xib screens.
2. The "Seconds" in countdown is now camel case.

There are a few minor UI issues that still need resolved.  The cancel button has a tint to match the button color, which makes the most sense.  However, design now wants it to be the color of underlined buttons and header text colors.  I am still figuring out the best way to proceed, and may need to create a new button type which is navigation icon button.  

The active step countdown label is not centered within the progress circle on some screen sizes.

Attached here is the kinetic tremor with PKU styling on iPhone SE and iPhone 7+

iPhone SE
KineticTremorPart1_SE
KineticTremorPart2_SE

iPhone 7+
KineticTremorPart1_7
KineticTremorPart2_7
KineticTremorPart3_7

### CODE REVIEW

File: MotorControl/MotorControl/Resources/Kinetic_Tremor.json [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+                                "reason":"Access to Motion and Fitness sensors is required to measure the phone's steadiness in your hand."
+                               }
+                            ],
+                 "image":{
+                     "type":"fetchable",
+                     "imageName":"FingerToNose9",
+                     "placementType":"topBackground"
+                 },
+                "videoResourceName": "instruction_video",
+                "videoBundleIdentifier": "org.sagebionetworks.MotorControl",
```
Comment (syoung-smallwisdom) [inline MotorControl/MotorControl/Resources/Kinetic_Tremor.json]: See comment in other PR about using a UIAction and resource transformer to access the URL for this video.

Additionally, please rename the url to be specific to the tremor activity.

File: MotorControl/MotorControl/Resources/Kinetic_Tremor.json [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+                             "text": "I can perform this activity with both hands.",
+                             "value": "both"
+                         }
+                     ]
+                 }]
+             },
+             {
+                "identifier":"eachHandInstructions",
+                "type":"instruction",
+                "title":"Each hand for 30 seconds",
```
Comment (syoung-smallwisdom) [inline MotorControl/MotorControl/Resources/Kinetic_Tremor.json]: So... In the previous step, the user said they only can do this with one had. Reorder to put this instruction *before* asking about their hands.

File: MotorControl/MotorControl/Resources/Kinetic_Tremor.json [hunk 3]
```diff
@@ -0,0 +1,174 @@
...
+             {
+                 "identifier":"howToTouchNoseInstructions",
+                 "type":"instruction",
+                 "title":"How to touch your nose",
+                 "text":"Hold your arm straight out beside you, then bend your elbow to bring the tip of your finger to the tip of your nose. Try to touch precisely if you can.",
+                 "image":{
+                     "type":"animated",
+                     "imageNames":["KineticHoldPhone-Left", "FingerToNose4", "FingerToNose5"],
+                     "placementType":"topBackground",
+                     "animationDuration": 3
```
Comment (syoung-smallwisdom) [inline MotorControl/MotorControl/Resources/Kinetic_Tremor.json]: Both here and in the sit down, you are using a 3 second timer. Is the cadence really that fast? 4/4 time is an easier rhythm for most people, yeah?

File: MotorControl/MotorControl/Resources/MotorControl.strings [hunk 4]
```diff
@@ -36,9 +36,9 @@
 
 "TAP_COUNT_LABEL" = "TAP COUNT";
 
-"ACTIVE_STEP_UNIT_LABEL" = "SECONDS";
+"ACTIVE_STEP_UNIT_LABEL" = "Seconds";
 
-"ACTIVE_STEP_UNIT_LABEL_SINGULAR" = "SECOND";
+"ACTIVE_STEP_UNIT_LABEL_SINGULAR" = "Second";
```
Comment (syoung-smallwisdom) [inline MotorControl/MotorControl/Resources/MotorControl.strings:41]: For consistency, since the design team wants to change this to capitalized, should also change "TAP COUNT" above.

**Review by syoung-smallwisdom — CHANGES_REQUESTED** [review]
Please add unit tests for this task, update access to video to use a UIAction *and* to hide the video if none available. Additionally, before submitting a PR, please smoke test other tasks that will be effected by the changes you are making here.

File: MotorControl/MotorControl/Resources/Kinetic_Tremor.json (hunk 3 above)
Comment (dephillipsmichael) [reply MotorControl/MotorControl/Resources/Kinetic_Tremor.json]: I don't think 4/4 means what you think it means

File: MotorControl/MotorControl/Resources/Kinetic_Tremor.json (hunk 2 above)
Comment (dephillipsmichael) [reply MotorControl/MotorControl/Resources/Kinetic_Tremor.json]: I was just following the example of the resting tremor which has this exact flow.  I feel like since it's never been a problem on any other app, we should just leave it as is.

Also, say we did move it in front of the which hand question... if they selected to only do it with one hand, that copy is still inaccurate.  It's unavoidable without custom dynamic copy which I think is overkill to fix a non-issue.

File: MotorControl/MotorControl/Resources/Kinetic_Tremor.json (hunk 1 above)
Comment (dephillipsmichael) [reply MotorControl/MotorControl/Resources/Kinetic_Tremor.json]: Will do

File: MotorControl/MotorControl/Resources/MotorControl.strings (hunk 4 above)
Comment (dephillipsmichael) [reply MotorControl/MotorControl/Resources/MotorControl.strings:41]: Done 👍

File: MotorControl/MotorControl/Resources/Kinetic_Tremor.json (hunk 2 above)
Comment (syoung-smallwisdom) [reply MotorControl/MotorControl/Resources/Kinetic_Tremor.json]: If you look at the text for this view, it talks about "you will do this with your right hand and then with your left hand." Seems to me that it makes more sense to put it before asking about which hand, yeah?

Additionally, it's not included in the "review instructions" and is more of an extension of the overview.

File: MotorControl/MotorControl/Resources/Kinetic_Tremor.json (hunk 3 above)
Comment (syoung-smallwisdom) [reply MotorControl/MotorControl/Resources/Kinetic_Tremor.json]: Fair enough. 4/4 time at 120 bpm.

What I mean is that no where in the text does it specify how fast to go, but a comfortable rate that people will naturally gravitate towards tends to be the "house" beat. 

I think you are probably animating it too fast.

File: MotorControl/MotorControl/Resources/MotorControl.strings (hunk 4 above)
Comment (syoung-smallwisdom) [reply MotorControl/MotorControl/Resources/MotorControl.strings:41]: thanks

**Comment by dephillipsmichael** [comment]
> Please add unit tests for this task, update access to video to use a UIAction _and_ to hide the video if none available. Additionally, before submitting a PR, please smoke test other tasks that will be effected by the changes you are making here.

I'm struggling to come up with unit tests for the task. All I see in MotorControlTests that is relevant to testing the tasks is NavigationTests.swift, and that thoroughly covers all the Kinetic Tremor navigation scenarios.

**Comment by dephillipsmichael** [comment]
Switched to video view action

File: MotorControl/MotorControl/Resources/Kinetic_Tremor.json [hunk 5]
```diff
@@ -0,0 +1,178 @@
...
+             {
+                "identifier":"sitDownInstruction",
+                "type":"instruction",
+                "title":"Sit down",
+                "text":"Find a spot to sit where you can have your feet resting flat on the floor.",
+                "image":{
+                    "type":"animated",
+                    "imageNames":["FingerToNose0", "FingerToNose1"],
+                    "placementType":"topBackground",
+                    "animationDuration": 4,
```
Comment (dephillipsmichael) [inline MotorControl/MotorControl/Resources/Kinetic_Tremor.json:97]: Switched to 4 second duration for you @syoung-smallwisdom also switched to repeat count of 1 as requested by design

File: MotorControl/MotorControl/Resources/Kinetic_Tremor.json (hunk 2 above)
Comment (dephillipsmichael) [reply MotorControl/MotorControl/Resources/Kinetic_Tremor.json]: Ok, switched it to before the hand question step for you.

File: .gitmodules [hunk 6]
```diff
@@ -1,3 +1,3 @@
 [submodule "Carthage/Checkouts/SageResearch"]
 	path = Carthage/Checkouts/SageResearch
-	url = https://github.com/Sage-Bionetworks/SageResearch.git
+	url = https://github.com/dephillipsmichael/SageResearch.git
```
Comment (syoung-smallwisdom) [inline .gitmodules:3]: Master needs to point at Sage.

**Review by syoung-smallwisdom — CHANGES_REQUESTED** [review]
Can you please point Cartfile back to sage? Thanks
